common-tool/project-arm.mk

```diff
@@ -45,6 +47,19 @@ elfloader: libcpio common FORCE
 		| while read line; do echo " [GEN_IMAGE] $$line"; done; \
 		exit $${PIPESTATUS[0]}
 
+# make pre-image IMAGE_DIR=~/sel4-tutorials/stage/arm/zynq7000/bin/hello-3
```

Comment:
Are we able to delete this pre-image rule?  It appears to be left over from testing.